README.md

```diff
@@ -59,22 +57,18 @@ after logging in to [AWS](https://login.oslo.kommune.no/auth/realms/AD/protocol/
 
 ```bash
 
-# Create a cluster. Format:
-# okctl create cluster <environment name> <AWS account ID>
-#
-# <environment name>    can be for instance "prod" or "test".
-# <AWS account ID>      is the account ID described in the above
-#
-# Example:
-$ okctl create cluster prod 123456789012
+# Scaffold a cluster. Format:
+# okctl scaffold cluster -f cluster.yaml
+# <edit cluster.yaml>
+# okctl apply cluster -f cluster.yaml
 ```
 
 Follow the instructions.
 
 When done, verify that you have a working cluster by running
 
 ```bash
-$ okctl venv
+$ okctl venv -c cluster.yaml
```

Comment:
Are we removing the concept of environment completely for the user?

Reply:
Basically, to be honest, I am not even sure why we had it there in the first place 🤷🏽

Reply:
I like this approach, I'm pretty pro users choosing how to do stuff like this